packages/sdk-pjs/src/ethTransfer/createContext.test.ts

```diff
@@ -1,43 +1,43 @@
 import { describe, it, expect, vi } from 'vitest'
-import { contextFactory } from '@snowbridge/api'
+import { Context } from '@snowbridge/api'
 import { AbstractProvider } from 'ethers'
 import { createContext } from './createContext'
 import type { Config } from '@snowbridge/api/dist/environment'
+import { ETHEREUM_RPC_URL } from '../consts'
 
 vi.mock('@snowbridge/api', () => ({
-  contextFactory: vi.fn()
+  Context: vi.fn().mockImplementation(() => ({}))
 }))
 vi.mock('ethers', () => ({
   AbstractProvider: class {}
 }))
 
 describe('createContext', () => {
-  it('creates a context with the correct structure', async () => {
+  it('creates a context with the correct structure', () => {
     const mockConfig = {
       BEACON_HTTP_API: 'http://beacon-api.test',
-      BRIDGE_HUB_URL: 'http://bridge-hub.test',
-      ASSET_HUB_URL: 'http://asset-hub.test',
+      ASSET_HUB_PARAID: 1000,
+      BRIDGE_HUB_PARAID: 2000,
       RELAY_CHAIN_URL: 'http://relay-chain.test',
       PARACHAINS: ['http://parachain1.test', 'http://parachain2.test'],
```

Comment:
Asset Hub and Bridge Hub are moved into the parachains list.
```suggestion
      PARACHAINS: ['http://bridge-hub.test', 'http://asset-hub.test'],
```
Also the type of parachains has changed from `string[]` to `{ [paraId: string]: string }`.

Reply:
I fixed it! thanks.